Extracting reference data (Isolation forest).ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        df = df.drop(["Date_of_observation", "Observation_ID", "User_id", "User_Tree_id", "Species_id", "State_name", "Species_name", "Year"], axis=1)
</pre><p>Lat/Long are currently not being removed. Could be interesting to experiment with this as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>